pkgs/top-level/all-packages.nix

```diff
@@ -30320,7 +30314,7 @@ with pkgs;
 
   atomPackages = dontRecurseIntoAttrs (callPackage ../applications/editors/atom { });
 
-  inherit (atomPackages) atom atom-beta;
+  inherit (atomPackages) atom;
```

Comment:
This is what's erroring, you removed the source for atom but not atom, only atom-beta

Reply:
Removed that patch, because there are some other packages that depend on expressions in the atom space, and I didn't figure out whether they are broken too or not.